### PR TITLE
Dev update RPC server description

### PR DESCRIPTION
**Pull request by jimmy121192**
update RPC server description and update packages versions

### CODE REVIEW

File: edgepi-dac.js [hunk 1]
```diff
@@ -13,7 +13,10 @@ module.exports = function (RED) {
         node.status({ fill: "green", shape: "dot", text: "input received" });
         try {
           gain = msg.gain || gain;
-          voltage = msg.payload || voltage;
+          voltage =
+            msg.payload && typeof msg.payload == "number"
+              ? msg.payload
+              : voltage;
           channel = msg.channel || channel;
```
Comment (maria-mz) [inline edgepi-dac.js]: why do we check just `msg.payload` and not other parts of the message like `msg.gain` and `msg.channel`?

Comment (jimmy121192) [reply edgepi-dac.js]: Code has been updated. But to answer your question with my old code:

`msg.gain` and `msg.channel` are already checked with || operator

**Review by farzadpanahi — APPROVED** [review]

**Review by farzadpanahi — DISMISSED** [review]

File: edgepi-dac.js [hunk 2]
```diff
@@ -4,17 +4,22 @@ module.exports = function (RED) {
...
+          if (msg.payload) {
+            voltage = msg.payload;
+          }
+          if (typeof msg.gain === "boolean") {
+            gain = msg.gain;
+          }
+          if (msg.channel) {
+            channel = msg.channel;
+          }
+
```
Comment (maria-mz) [inline edgepi-dac.js:22]: If these `if` conditions aren't met, should we be continuing with the rest of the code?

**Review by maria-mz — CHANGES_REQUESTED** [review]

File: edgepi-dac.js (hunk 2 above)
Comment (jimmy121192) [reply edgepi-dac.js:22]: yes. it is

**Review by maria-mz — APPROVED** [review]

**Pull request merged**